brunch-config.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  paths: {
+    watched: ['scss-c2s']
+  },
+  conventions: {
+    ignored: 'scss-c2s/c2s-ie.scss'
+  },
+  overrides: {
+    development: {
+      sourceMaps: false,
```

Comment:
Why are source maps not used in development?

Reply:
I don't use them. But I guess "this is for everyone" So i'll put them back.

Reply:
Don't bother making any changes. See https://github.com/nhsuk/profiles/pull/250#pullrequestreview-70197434